Tests/Common/Util/WhoCalledMeTests.cs

```diff
@@ -22,6 +23,7 @@ namespace QuantConnect.Tests.Common.Util
     public class WhoCalledMeTests
     {
         [Test]
+        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
Is this required?

Reply:
Currently, the test passes so no, we don't need it. But in theory yes, it could start inlining and make the test fail